Dockerfile

```diff
@@ -142,6 +142,10 @@ EXPOSE 8888
 WORKDIR /data/
 RUN fix-permissions /data
 
+USER $NB_UID
+CMD chmod -R 777 /data

```

Comment:
shouldn't this be done under root?

Reply:
the fix-permissions line above should make nb_uid owner so I think it's ok

Reply:
might be worth a comment then, cause it looks a bit weird :)